src/sentry/incidents/models.py

```diff
@@ -560,6 +574,7 @@ class AlertRuleTriggerAction(AbstractNotificationAction):
     """
 
     __include_in_export__ = True
+    __relocation_scope__ = RelocationScope.Global
```

Comment:
Alert rules are going to be tricky with relocations. Notification delivery targets are often provided by integrations which we cant move. However, I don't know that moving alert rules without delivery targets is overly useful either.

Reply:
Noted. My current plan is to (at some point) add an a "variable" relocation scope that resolves whether the model is included in the relocation on a per-mode-instance, rather than per-model, basis. There are relatively few such models included today though, just the descendants `AbstractNotificationAction` and `UserOption`. I think keeping the former `Global` and the latter `User` should be okay for a first pass.